source/refined-github.css

```diff
@@ -35,15 +35,6 @@
 	display: none !important;
 }
 
-/* Remove user avatar from new comment form on threads */
-.review-thread-reply .avatar {
-	display: none;
-}
-
-.review-thread-reply .review-thread-reply-button {
-	margin-left: 0;
-}
-
```

Comment:
Apparently forgotten in [`f099d37` (#1986)](https://github.com/refined-github/refined-github/pull/1986/commits/f099d3737111e9d04917267f9ebb2a4efc057368)